unit-tests/jvm/src/test/scala/utils/Platform.scala

```diff
@@ -25,4 +25,6 @@ object Platform {
   private val osNameProp = System.getProperty("os.name")
   final val isFreeBSD = osNameProp.equals("FreeBSD")
   final val isWindows = osNameProp.toLowerCase.startsWith("windows")
+  final val is32BitPlatform = false
```

Comment:
Hmm, how would we handle tests that are shared between JVM and native, but don't work on 32-bit platforms? The idea here is that we just stub the JVM so that tests can compile with this value.

In the helper assume method, I'll make it so that on the JVM the assumption never fails so that we never skip a test.

Reply:
Ok, that seems reasonable. There are 4 shared tests that use that value, for 3 of them `DoubleTest`, `FloatTest` and `RandomTest` we should be able to provide conditionally expected behaviour that should pass on both JVM and Native. 

We should not skip these tests on the JVM, instead, we can use system property `System.getProperty("sun.arch.data.model")` (returns `32`, `64` or `unknown`) to check used architecture or other solutions, eg. described in this thread https://stackoverflow.com/questions/2062020/how-can-i-tell-if-im-running-in-64-bit-jvm-or-32-bit-jvm-from-within-a-program